pepper-apis/src/main/java/org/broadinstitute/ddp/util/SecretUtil.java

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.ddp.util;
+
+import java.io.IOException;
+
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import org.broadinstitute.ddp.constants.ConfigFile;
+import org.broadinstitute.ddp.exception.DDPException;
+
+public class SecretUtil {
+    public static Config getConfigFromSecret(Config mainCfg, String secretName) {
+        try (SecretManagerServiceClient client = SecretManagerServiceClient.create()) {
+            SecretVersionName versionName = SecretVersionName.of(mainCfg.getString(ConfigFile.GOOGLE_PROJECT_ID), secretName, "latest");
```

Comment:
Looks like GCP project id is the only thing we need here. Can we just pass that in instead of a whole config object?